samples/pyarrow/append_rows_with_arrow_test.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from . import append_rows_with_arrow
+
+
+def test_append_rows_with_arrow(capsys, project_id, dataset):
```

Comment:
Do we want to do a deeper validation here, like running a SELECT query against the destination that validates the data arrived?

Reply:
Yeah, that would be a lot better! I will add the verification section.

Reply:
Added verification.